apps/bixarena/app/src/model/models_api.py

```diff
@@ -0,0 +1,124 @@
+"""
+Simple FastAPI endpoint to serve model data
+Single endpoint: GET /models
+Includes database access functions - no external dependencies
+"""
+
+import subprocess
+from fastapi import FastAPI, HTTPException
+from typing import Any
+import uvicorn
+
+app = FastAPI(title="BixArena Models API", version="1.0.0")
+
+
+def _run_query(query):
+    """Run PostgreSQL query"""
+    cmd = [
+        "psql",
+        "-h",
+        "bixarena-postgres",
+        "-p",
+        "21000",
+        "-U",
+        "postgres",
+        "-d",
+        "bixarena",
+        "-t",
+        "-A",
+        "-F",
+        "|",
+        "-c",
+        query,
+    ]
```

Comment:
Fetching data using this CLI is not a good design. Interacting with a database should be done with a database client library.